src/components/molecules/Post/Post.jsx

```diff
@@ -0,0 +1,65 @@
+import { useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { useNavigate } from 'react-router-dom';
+// import { Form } from 'formik';
+import { Text, ImageComponent } from '../..';
+import { routes } from '../../../routes/Routes';
+import { Icon } from '../../atoms/IconComponent/Icon';
+import { dateFormat } from '../../../utils/format';
+
+import './Post.scss';
+
+export const Post = ({ children, date, name, avatar, profileID }) => {
+    const navigate = useNavigate();
+    const redirect = useCallback(() => {
+        navigate(routes.Profile);
```

Comment:
pls add TODO comment that we need to pass an profile ID here to route

Reply:
done